packages/Button/stories/variations/Variations.js

```diff
@@ -1,324 +1,23 @@
 import React from "react";
-import { action } from "@storybook/addon-actions";
-import styled from "styled-components";
-import { Rule, Tagline, breaklines, Gap } from "storybook/assets/styles/common.styles";
-import tokens from "@paprika/tokens";
-import Popover from "@paprika/popover";
-import PlusIcon from "@paprika/icon/lib/Add";
-import InfoIcon from "@paprika/icon/lib/InfoCircle";
+import { Rule, Tagline, breaklines } from "storybook/assets/styles/common.styles";
 import { ButtonStory } from "../Button.stories.styles";
-import Button from "../../src";
-
-const DarkBackground = styled.span`
-  background: ${tokens.color.black};
-  display: inline-block;
-  margin-left: ${tokens.space};
-  padding: ${tokens.space};
-`;
-
-function clickHandler() {
-  action("Clicked a button")();
-}
-
-const renderTriggerWithTooltip = () => {
-  const content = "Tooltip text example";
-  return (
-    <Popover align="top" isDark isEager offset={8} shouldKeepFocus>
-      <Popover.Trigger tabIndex="-1">
-        <PlusIcon />
-      </Popover.Trigger>
-      {content && (
-        <>
-          <Popover.Content key={content}>
-            <Popover.Card>{content}</Popover.Card>
-          </Popover.Content>
-          <Popover.Tip />
-        </>
-      )}
-    </Popover>
-  );
-};
+import ButtonVariations from "./ButtonVariations";
+import LinkButtonVariations from "./LinkButtonVariations";
+import IconButtonVariations from "./IconButtonVariations";
+import CloseButtonVariations from "./CloseButtonVariations";
 
 export default function Variations() {
   return (
     <ButtonStory>
       <Tagline>So many different kinds of Buttons!</Tagline>
```

Comment:
Let's add a `<Heading level={1} displayLevel={2} isLight>Variations of <Button></Heading>` here.

Reply:
Or maybe it should go in the `ButtonVariations.js` file.  In that case we can move this `<Tagline>` too (or just get rid of it).